### ISSUE TITLE
Tiger 2015 Edges

### ISSUE THREAD

**Issue by d3mon187** [twain47/Nominatim#335]
I just installed Nominatim on a CentOS server and have reached the point in the wiki install instructions for importing tiger edge data.  I noticed Tiger2015 is out, and was wondering if it's compatible with the current stable Nominatim release?  I did see that PostGIS only shows compatibility for vs 2.2.x, which leads me to think it isn't.  Thanks!

**Comment by mtmail** [comment]
The file format didn't change between 2014 and 2015. Even in a file with 50.000 entries less than 100 entries are different.

In the stable Nominatim release you still need to use the `--parse-tiger-2011` parameter (the format changed between 2010 and 2011). That's cleaned up in the latest version in git.

**Comment by d3mon187** [comment]
Ok, awesome, thanks.  I've now run the 2014 import, but I'd still like to use the 2015 data since my program depends on the newest locations.  Can I just run the import steps again with the 2015 data, or do I need to worry about duplicates or anything?  Thanks again!

**Comment by mtmail** [comment]
During import all data gets placed in a table called `location_property_tiger_import`. As last step the `location_property_tiger` get created (dropped if already exists) and indices get (re)created. So it's safe to just import the 2015 on top of 2014.

**Comment by d3mon187** [comment]
Excellent, I'll give it a go once the 2014 is finished.  Appreciate the help!

**Comment by TroyGitHub** [comment]
I know this issue might be closed, but I want to give some comments. I read the Tiger data specification and it mentioned the best source for geocoding is the "addrfeat.shp" file instead of "Edge.shp", are there any reasons to use edge.shp?

See table 5: Address range product comparison table
http://www2.census.gov/geo/pdfs/maps-data/data/tiger/tgrshp2015/TGRSHP2015_TechDoc_Ch4.pdf

**Comment by lonvia** [comment]
The address range data didn't exist when Tiger support was implemented, so that would count as feature request (i.e. somebody needs to program it). Feel free to open a new issue. This issue should be closed as you correctly noted.
